src/modules/ie-objects/ie-objects.consts.tsx

```diff
@@ -97,6 +89,17 @@ export const METADATA_EXPORT_OPTIONS = (): MenuItemInfo[] => [
 	},
 ];
 
+export const NEWSPAPER_DOWNLOAD_OPTIONS = (): MenuItemInfo[] => [
```

Comment:
i prefer functions to start with a verb, so its more clear that its a function
You can do the same for METADATA_EXPORT_OPTIONS => GET_METADATA_EXPORT_OPTIONS
```suggestion
export const GET_NEWSPAPER_DOWNLOAD_OPTIONS = (): MenuItemInfo[] => [
```